src/main/java/ToDo.java

```diff
@@ -0,0 +1,12 @@
+public class ToDo extends Task {
+
+
```

Comment:
Hmm, small issue, but for consistency sake and general neatness of the code, should these empty lines be removed?

Reply:
Noted, thank you!